rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/dto/JobInfoData.java

```diff
@@ -228,8 +238,8 @@ public String toString() {
                ", numberOfRunningTasks=" + numberOfRunningTasks + ", numberOfFinishedTasks=" + numberOfFinishedTasks +
                ", numberOfFailedTasks=" + numberOfFailedTasks + ", numberOfFaultyTasks=" + numberOfFaultyTasks +
                ", numberOfInErrorTasks=" + numberOfInErrorTasks + ", priority=" + priority + ", jobOwner='" + jobOwner +
-               '\'' + ", toBeRemoved=" + toBeRemoved + ", genericInformation=" + genericInformation + ", variables=" +
-               variables + '}';
+               "', projectName='" + projectName + "', toBeRemoved=" + toBeRemoved + ", genericInformation=" +
```

Comment:
', projectName=' => , projectName=
', toBeRemoved => , toBeRemoved

Reply:
project name can contain spaces, this is why we put it inside quotes, toBeRemoved does not contain spaces